cloudbuild.yaml

```diff
@@ -43,15 +48,20 @@ steps:
     #!/bin/sh
     set -o errexit
     set -o xtrace
-    docker tag bazel/base:static_debian9          gcr.io/$PROJECT_ID/static:${COMMIT_SHA}
-    docker tag bazel/base:static_debian9          gcr.io/$PROJECT_ID/static:latest
-    docker tag bazel/base:static_debian9          gcr.io/$PROJECT_ID/static-debian9:${COMMIT_SHA}
-    docker tag bazel/base:static_debian9          gcr.io/$PROJECT_ID/static-debian9:latest
-    docker tag bazel/base:static_debian10         gcr.io/$PROJECT_ID/static-debian10:${COMMIT_SHA}
-    docker tag bazel/base:static_debian10         gcr.io/$PROJECT_ID/static-debian10:latest
-    docker tag bazel/base:static_nonroot_debian9  gcr.io/$PROJECT_ID/static:nonroot
-    docker tag bazel/base:static_nonroot_debian9  gcr.io/$PROJECT_ID/static-debian9:nonroot
-    docker tag bazel/base:static_nonroot_debian10 gcr.io/$PROJECT_ID/static-debian10:nonroot
+
+    docker tag bazel/base:static_root_amd64_debian9     gcr.io/$PROJECT_ID/static:${COMMIT_SHA}
+    docker tag bazel/base:static_root_amd64_debian9     gcr.io/$PROJECT_ID/static:latest
+    docker tag bazel/base:static_root_amd64_debian9     gcr.io/$PROJECT_ID/static-debian9:${COMMIT_SHA}
+    docker tag bazel/base:static_root_amd64_debian9     gcr.io/$PROJECT_ID/static-debian9:latest
+    docker tag bazel/base:static_root_amd64_debian10    gcr.io/$PROJECT_ID/static-debian10:${COMMIT_SHA}
+    docker tag bazel/base:static_root_amd64_debian10    gcr.io/$PROJECT_ID/static-debian10:latest
+    docker tag bazel/base:static_nonroot_amd64_debian9  gcr.io/$PROJECT_ID/static:nonroot
+    docker tag bazel/base:static_nonroot_amd64_debian9  gcr.io/$PROJECT_ID/static-debian9:nonroot
+    docker tag bazel/base:static_nonroot_amd64_debian10 gcr.io/$PROJECT_ID/static-debian10:nonroot
+
```

Comment:
Wanna sprinkle some `docker manifest create` in here? :D

Reply:
cc @jonjohnsonjr 

as I was hoping he'd do the multi-arch stitching, but given how this is publishing the images, maybe rules_docker work is wholly unnecessary? Regardless, out of scope for this change 😉

Reply:
Looks like I still need to add these to the images like to publish though.

Reply:
Also, I have a PoC where I just enabled the experimental docker features and published a manifest list with these from GCB 🤩 , so expect another PR on top of this soon.  I'll fix `images:` here first, but I don't think we can use `images:` to publish the manifest lists, unfortunately.

Reply:
ok, added to `images:` going to start on a PR that publishes manifest lists next.